idle_sweep_test.go

```diff
@@ -88,6 +107,21 @@ func numConnections(ch *Channel) int {
 	return count
 }
 
+func numExchanges(ch *Channel) int {
+	var num int
+	rootPeers := ch.RootPeers().Copy()
+	for _, p := range rootPeers {
+		state := p.IntrospectState(&IntrospectionOptions{IncludeExchanges: true})
+		for _, c := range state.InboundConnections {
+			num += len(c.InboundExchange.Exchanges) + len(c.OutboundExchange.Exchanges)
+		}
+		for _, c := range state.OutboundConnections {
+			num += len(c.InboundExchange.Exchanges) + len(c.OutboundExchange.Exchanges)
+		}
```

Comment:
The `IncludeExchanges` is used to get detailed information about each exchange in `.Exchanges`, but in our case, we only need the count which is always set in `.Count`, so this can be simplified a little,
```suggestion
		state := p.IntrospectState(nil)
		for _, c := range state.InboundConnections {
			num += c.InboundExchange.Count + c.OutboundExchange.Count
		}
		for _, c := range state.OutboundConnections {
			num += c.InboundExchange.Count + c.OutboundExchange.Count
		}
```